src/DDTrace/Util/Versions.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace DDTrace\Util;
+
+/**
+ * Utility functions to handle version numbers and matching.
+ */
+final class Versions
+{
+    /**
+     * @param string $version
+     * @return bool
+     */
+    public static function phpVersionMatches($version)
+    {
+        return self::versionMatches($version, phpversion());
```

Comment:
Just to prevent the function call overhead. :)
```suggestion
        return self::versionMatches($version, PHP_VERSION);
```